frontend/js/src/common/brainzplayer/SpotifyPlayer.tsx

```diff
@@ -219,14 +219,16 @@ export default class SpotifyPlayer
     retryCount = 0
   ): Promise<void> => {
     const { device_id } = this.state;
-    const { handleError } = this.props;
+    const { handleError, onTrackNotFound } = this.props;
     if (retryCount > 5) {
       handleError("Could not play Spotify track", "Playback error");
+      onTrackNotFound();
```

Comment:
Unrelated to this PR, but I think this function call was missing and resulted in BrainzPlayer being stuck if there was a failure playing a track, instead of trying another source or skipping the track.